docs/oauth2-oidc/get-started/index.mdx

```diff
@@ -0,0 +1,19 @@
+---
+id: index
+title: OAuth2 & OpenID Connect
+sidebar_label: Get started
+---
+
+This guide covers the essential setup and implementation details for adding OAuth2 authorization flows to your application.
+
+## This guide is for you if:
+
+1. You want to implement OAuth2 authorization in your applications
+2. You want to quickly setup Ory's OAuth2 server
+
```

Comment:
There are more use cases here:

* Customer is migrating from Auth0 or another OAuth2/OpenID Connect-based authentication solution
* Customer is using a standard library for auth that uses OAuth2 / OpenID Connect (NextAuth, Auth.js, Passport.js, ...)